kustomize/commands/edit/add/configmap_test.go

```diff
@@ -4,13 +4,19 @@
 package add
 
 import (
+	"fmt"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/kustomize/api/provider"
+	"sigs.k8s.io/kustomize/kustomize/v5/commands/internal/kustfile"
+
 	"github.com/stretchr/testify/assert"
 	"sigs.k8s.io/kustomize/api/kv"
 	"sigs.k8s.io/kustomize/api/loader"
 	valtest_test "sigs.k8s.io/kustomize/api/testutils/valtest"
 	"sigs.k8s.io/kustomize/api/types"
+	testutils_test "sigs.k8s.io/kustomize/kustomize/v5/commands/internal/testutils"
```

Comment:
very nit: use testutils instread of testutils_test, feels strange to see two test in one word..
other lgtm

/lgtm

Reply:
Because this is named as such in every usage, I'll create a separate PR to address it (and rename across the project for consistency).

Reply:
I opened #5343 for discussing this change but there are many more packages that are imported with a similar naming pattern.